pythermalcomfort/models/work_capacity.py

```diff
@@ -0,0 +1,243 @@
+from typing import Union
+
+import numpy as np
+
+from pythermalcomfort.classes_input import (
+    WorkCapacityHothapsInputs,
+    WorkCapacityStandardsInputs,
+)
+from pythermalcomfort.classes_return import WorkCapacity
+
+
+def workcapacity_dunne(wbgt: Union[float, list[float]], intensity: str) -> WorkCapacity:
+    """
+    Estimate work capacity due to heat based ISO standards as described by Dunne et al
+
+    Estimates the amount of work that will be done at a given WBGT and
+    intensity of work as a percent. 100% means work is unaffected by heat. 0%
+    means no work is done.
+
+    This is based upon NIOSH safety standards. See:
+    Dunne JP, Stouffer RJ, John JG. Reductions in labour capacity from heat
+    stress under climate warming. Nature Climate Change. 2013 Jun;3(6):563–6.
+
+    Heavy intensity work is sometimes labelled as 400 W, but this is only
+    nominal. Medium work is assumed to be half as much as heavy intensity, and
+    light half as much as medium.
+
+    Parameters
+    ----------
+    wbgt : float or list of floats
+        Wet bulb globe temperature, [°C].
+    intensity : str
+        Which work intensity to use for the calculation, choice of "heavy",
+        "med" or "light".
+
+    Returns
+    -------
+    WorkCapacity
+        A dataclass containing the work capacity. See
+        :py:class:`~pythermalcomfort.classes_return.WorkCapacity` for more details. To access the
+        `capacity` value, use the `capacity` attribute of the returned `WorkCapacity` instance, e.g.,
+        `result.capacity`.
+
+    """
+    # Validate inputs
+    WorkCapacityHothapsInputs(wbgt=wbgt, intensity=intensity)
+
+    wbgt = np.array(wbgt)
+    capacity = np.clip((100 - (25 * (np.maximum(0, wbgt - 25)) ** (2 / 3))), 0, 100)
+    if intensity == "heavy":
+        pass
```

Comment:
in the function inputs please set heavy as default value since this is what the code does.